3_Implementation/documentation/html/search/typedefs_0.js

```diff
@@ -0,0 +1,15 @@
+var searchData=
+[
+  ['unity_5fdouble_895',['UNITY_DOUBLE',['../unity__internals_8h.html#ac3bdb18f38e23a4343af34a7f4e0d4a0',1,'unity_internals.h']]],
+  ['unity_5ffloat_896',['UNITY_FLOAT',['../unity__internals_8h.html#ac4a36d11e8a7b250f2413ebb22e3092c',1,'unity_internals.h']]],
+  ['unity_5ffloat_5ftrait_5ft_897',['UNITY_FLOAT_TRAIT_T',['../unity__internals_8h.html#a1b89a0178ea753186faf9f3dc271e2ba',1,'unity_internals.h']]],
+  ['unity_5fint_898',['UNITY_INT',['../unity__internals_8h.html#ab304e341ff16e7076e9d06cc9c64c2a7',1,'unity_internals.h']]],
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/2015pushkar/C-mini-Project-303459-LTTS/pullRequest?prid=7744834)
```suggestion
  ["unity_5fint_898",['UNITY_INT',['../unity__internals_8h.html#ab304e341ff16e7076e9d06cc9c64c2a7',1,'unity_internals.h']]],
```